packages/web-frontend/src/historyNavigation/utils.js

```diff
@@ -22,8 +22,9 @@ import {
 export const createLocation = params => {
   const { userPage } = params;
   if (userPage) {
-    // TODO: Userpage logic to come in future PR
-    return { pathname: `/${userPage}` };
+    // Userpage locations are created by the backend,
+    // this is good enough for here
+    return { pathname: `/${userPage}`, search: {} };
```

Comment:
Do you agree with this, or should it throw an error?

Reply:
Or possibly it should just return all the search params blindly.

Reply:
Since in other cases `search` just includes params that front-end is aware of, I'd say this is fine as is!

Eg for `https//tupaia.org/explore/TO/dashboardId?overlay=overlayId&random=stuff` , `{ random: 'stuff' }` will not be includes in `search`